src/plugins/kibana/public/settings/sections/indices/add_data_steps/paste_samples_step/paste_samples_step.html

```diff
@@ -0,0 +1,11 @@
+<h2><em>Provide some sample logs.</em>
+Paste in one or more lines from the file you intend to tail. We'll use these samples in the following steps to help
+you build an ingest pipeline and configure a Kibana index pattern. Log lines can be raw strings or
+formatted as JSON. If your logs are raw strings but you intend to use
+<a target="_window" href="https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields.html">Filebeat's metadata</a>,
```

Comment:
Did this make it past your extermination of external links for a reason? Is this url correct/future proof?